packages/sdks/mitosis.config.cjs

```diff
@@ -825,78 +825,56 @@ const ANGULAR_COMPONENT_REF_UPDATE_TEMPLATE_SSR = () => ({
   },
 });
 
-const ANGULAR_MARK_SAFE_INNER_HTML = () => ({
-  code: {
-    post: (code, json) => {
-      if (['BuilderText', 'BuilderEmbed', 'CustomCode'].includes(json.name)) {
-        code =
-          `import { DomSanitizer } from "@angular/platform-browser";\nimport { ChangeDetectionStrategy } from "@angular/core";\n` +
-          code;
-
-        // add changeDetection: ChangeDetectionStrategy.OnPush
-        const changeDetectionIndex = code.indexOf('selector:');
-        if (changeDetectionIndex !== -1) {
-          code =
-            code.slice(0, changeDetectionIndex) +
-            `changeDetection: ChangeDetectionStrategy.OnPush,\n` +
-            code.slice(changeDetectionIndex);
-        }
-
-        // add constructor with sanitizer
-        const constructorIndex = code.indexOf('constructor');
-        if (constructorIndex === -1) {
-          // not found
-          const ngOnChangesIndex = code.indexOf('ngOnChanges');
-          code =
-            code.slice(0, ngOnChangesIndex) +
-            `constructor(protected sanitizer: DomSanitizer) {}\n` +
-            code.slice(ngOnChangesIndex);
-        } else {
-          throw new Error(
-            'constructor found which should not be here. If you see this, please fix the ANGULAR_TEXT_MARK_SAFE_HTML Plugin.'
-          );
-        }
-
-        const variableName = code.match(/\[innerHTML\]="([^"]+)"/)?.[1];
-        if (variableName) {
-          code = code.replace(
-            `[innerHTML]="${variableName}"`,
-            `[innerHTML]="sanitizer.bypassSecurityTrustHtml(${variableName})"`
-          );
-        }
-      }
-      return code;
-    },
-  },
-});
-
 /**
  * Angular allows DOM manipulation in `ngAfterViewInit` hook.
  * This plugin moves the DOM code from `ngOnInit` to `ngAfterViewInit` hook.
  */
-
 const ANGULAR_MOVE_DOM_MANIPULATION_CODE_TO_AFTERVIEWINIT = () => ({
   json: {
     post: (json) => {
-      if (json.name === 'BuilderVideo') {
+      if (['BuilderVideo', 'CustomCode', 'BuilderEmbed'].includes(json.name)) {
+        let hooks = {};
+
+        if (json.name === 'BuilderVideo') {
+          hooks.ngAfterViewInit = json.hooks.onMount[0];
+        }
+        if (json.name === 'CustomCode') {
+          hooks.ngAfterViewInit = json.hooks.onMount[0];
+          hooks.ngAfterViewChecked = json.hooks.onUpdate[0];
+        }
+        if (json.name === 'BuilderEmbed') {
+          hooks.ngAfterViewChecked = json.hooks.onUpdate[0];
+        }
+
         json.compileContext = {
           angular: {
-            hooks: {
-              ngAfterViewInit: json.hooks.onMount[0],
-            },
+            hooks,
           },
         };
 
-        json.compileContext.angular.hooks.ngAfterViewInit.code =
-          json.compileContext.angular.hooks.ngAfterViewInit.code
-            .replace('props.', 'this.')
-            .replace('state.', 'this.');
+        if (hooks.ngAfterViewInit) {
```

Comment:
We'll probably want to move that to mitosis too eventually: process any `json.compileContext.angular.hooks` that are provided by plugins.

we'd have to do that after `json.post` plugins are ran in the angular generator

Reply:
yup, created a ticket for this here:https://builder-io.atlassian.net/browse/ENG-8482